templates/Pages/rules_committee.php

```diff
@@ -16,8 +16,23 @@
     <h3>Committee</h3>
 
     <p>
-        The IO500 Committee, as of October 2020, consists of the three founding members (in alphabetical order), John Bent, Julian Kunkel, and Jay Lofstead, as well as two additional members, Andreas Dilger and George Markomanolis, who were a active early community participants and joined the committee subsequently.
+        The current <b>IO500 Committee<b>, consists of the following members (in alphabetical order):
     </p>
+    <ul>
+      <li>Andreas Dilger</li>
+      <li>Dean Hildebrand</li>
+      <li>Julian Kunkel</li>
+      <li>Jay Lofstead</li>
+      <li>George Markomanolis</li>
```

Comment:
I don't think we should list the Committee members in multiple places.  That just makes it easier to become inconsistent.  Currently, the members are listed in this page - `rules_committee.php`, in `rules_membership.php`, and Issue #50 is also asking for the members to be listed in `steering.php`.

I think the Committee should be listed in one place, probably `steering.php`, and the "rules" and "membership" should not really depend on the current membership itself.

Reply:
Good remark. I unified this in the new patch.